src/engine/runtime.js

```diff
@@ -1228,6 +1242,16 @@ class Runtime extends EventEmitter {
     }
 
     /**
+     * Report that a new target has been created, possibly by cloning an existing target.
+     * @param {Target} newTarget - the newly created target.
+     * @param {Target} [sourceTarget] - the target used as a source for the new clone, if any.
+     * @fires Runtime#targetWasCreated
+     */
+    fireTargetWasRemoved (newTarget, sourceTarget) {
+        this.emit('targetWasRemoved', newTarget, sourceTarget);
```

Comment:
Looks like the comment & arg list need to be updated here

Reply:
Good catch. Fixed in 
fa2db05